test/graph_rewrites/op_by_op_capability_test.cc

```diff
@@ -62,8 +62,7 @@ TEST(OpByOpCapability, Backend) {
   ASSERT_OK(BackendManager::GetCurrentlySetBackendName(&ng_backend_type));
 
   // Map with all the backends, and what the boolean is_supported should be
-  std::map<std::string, bool> backend_map{
-      {"CPU", true}, {"INTERPRETER", true}, {"NOP", false}};
+  std::map<std::string, bool> backend_map{{"CPU", true}, {"INTERPRETER", true}};
```

Comment:
removed NOP case here because we do not have static library available for this backend, so we cannot register NOP with static builds - so this test will fail.